fuse/plugins/truth_information/peak_truth.py

```diff
@@ -151,9 +213,12 @@ def compute(self, interactions_in_roi, propagated_photons, peaks):
                     )
                     photons_per_cluster_s2 = photons_per_cluster
 
-                    # Get the number of photons from delayed electrons
-                    photon_cut &= photons_in_peaks[i]["cluster_id"] < 0
-                    result["pi_photons_in_peak"][i] = np.sum(photon_cut)
+            # The raw area is defined by the number of PEs from the corresponding
+            # scintillation. Question: should we care mis-classification here?
+            if peak_type == 1:
+                result["raw_area_truth"][i] = result["s1_photoelectrons_in_peak"][i]
+            elif peak_type == 2:
+                result["raw_area_truth"][i] = result["s2_photoelectrons_in_peak"][i]
```

Comment:
I think this definition is not consistent with the one needed for the peak bias study. Also, I would like to keep this independent of the `peak_type`. For the area of a peak it does not matter if the same number of photons are from the S1 or the S2 branch of the simulation.

Reply:
Just saw that @dachengx also commented on the definition of the `raw_area`.

Reply:
I am asking for a change is because in appletree the bias is applied on the number of PEs (integer) to get the area (float), so my understanding is that the bias should be defined as `peak_area / number_of_PEs - 1`. If you think we should also keep raw_area's definition as it was, I can convert it back. But in the bias study I think we should change `raw_area_truth` to the number of corresponding PEs.